WordPress/Classes/ViewRelated/Blog/SharingAccountViewController.swift

```diff
@@ -12,6 +12,15 @@ import WordPressShared
     @objc var immutableHandler: ImmuTableViewHandler!
     @objc var delegate: SharingAccountSelectionDelegate?
 
+    lazy var noResultsViewController: NoResultsViewController = {
```

Comment:
Would it make sense to declare this var as `private`? Or do we really need it to be fully mutable?